aws/util.sh

```diff
@@ -5,7 +5,40 @@
 # All providers are expected to support the following functions:
 #	util::get_instance_info
 #	util::copy_file
-#
+
+# Helpers #
+# These functions implement lower leverl operations required for deploying
+# and destroying Amazon Web Service EC2 instances.  These functions should be
+# considered private, scoped to the provider.
+
+function __run_instance() {
+	# TODO parameterize options values
+	aws ec2 run-instances \
+	--count 2 \
+	--image-id ami-2fb42b39 \
+	--instance-type t2.large \
+	--key-name libra \
+	--subnet-id subnet-cf57c596 \
+	--tag-specifications \
+	'ResourceType=instance,Tags=[{Key=Name,Value=jcope-test}]' \
+	'ResourceType=volume,Tags=[{Key=Name,Value=jcope-test}]'
+}
+
+# Generic Utilities
```

Comment:
This comment block is fine but should be moved to the beginning of the file and merged with what is already there.

Reply:
The functions this comment block describes are at the bottom of the file, after the provider specific operations.  Above them are the provider's helper functions.  So splitting them apart doesn't seem like it would improve readability.